### PR TITLE
[ADD][10.0][TS0002RWU_TS02][BR000799][T21173] Added l10n_taiwan module

### PR DESCRIPTION
**Pull request by ghost**
@elicoidal Added l10n_taiwan localization module.

### CODE REVIEW

**Review by elicoidal — COMMENTED** [review]
Can you prepare a README file?
Besides I will create the basic repo structure for tests

**Comment by elicoidal** [comment]
@jbeficent I fixed the CLA in the ERP but still have a warning. Any idea?

**Comment by elicoidal** [comment]
@BipinRathod up?

**Comment by martintorresvm** [comment]
> Can you prepare a README file?
Besides I will create the basic repo structure for tests

@BipinRathod Any update on this topic?. Need help?

**Comment by ghost** [comment]
@victormartinelicocorp @elicoidal  I have added readme.rst file.

**Review by elicoidal — APPROVED** [review]
@victormartinelicocorp

File: l10n_taiwan/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+# © 2017 - Shanghai SITA Waste Treatment Services Company Limited.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    'name': 'l10n Taiwan',
+    'version': '10.0.1.0.0',
+    'summary': """This module adds state data for Taiwan Localization.""",
+    'author': 'Elico Corp',
```
Comment (martintorresvm) [inline l10n_taiwan/__manifest__.py]: @BipinRathod Append `Odoo Community Association (OCA)` to the `author`

**Review by martintorresvm — CHANGES_REQUESTED** [review]

File: l10n_taiwan/__manifest__.py (hunk 1 above)
Comment (ghost) [reply l10n_taiwan/__manifest__.py]: @victormartinelicocorp  Added Odoo Community Association (OCA) to the author

**Review by martintorresvm — APPROVED** [review]

**Pull request merged**